strongbox-security/strongbox-user-management/src/test/java/org/carlspring/strongbox/users/service/UserServiceTest.java

```diff
@@ -240,61 +240,6 @@ public void testThatUserNameIsUnique()
                 Collectors.toList()).size(), CoreMatchers.equalTo(1));
     }
 
-    @Test
-    public void testPrivilegesProcessingForAccessModel()
```

Comment:
I think this test will have to be re-worked. Perhaps you can create a new user and assign them the priviledges required below and improve it a bit. I don't think we should drop it.